format/FormatNexusJungfrauExt.py

```diff
@@ -0,0 +1,123 @@
+import h5py
+import numpy as np
+
+from xfel.util.jungfrau import pad_stacked_format
```

Comment:
Since `dxtbx` user may not have xfel configured please can we wrap this in a try / except?

Reply:
sounds good!